cms/jinja2/templates/base.html

```diff
@@ -56,10 +57,54 @@
     }
 }
 %}
+
 {# fmt:on #}
 {% block head %}
     <link rel="stylesheet" href="{{ static('css/main.css') }}">
-{% endblock %}
+
+  <!-- Google Tag Manager -->
+    <script>
```

Comment:
suggestion: wrap this and the other related bits in a `{% if GOOGLE_TAG_MANAGER_CONTAINER_ID %}` conditional. While not necessary, I think it is good practice, especially if one prefers to turn this off during local development

Reply:
I might add to that and suggest `{% if GOOGLE_TAG_MANAGER_CONTAINER_ID and not request.is_preview %}`

Reply:
That is a great shout, Andy. 👍🏼

Reply:
Assumption made here: I'm guessing you're meaning that the banner should not be displayed in the preview mode so I'm wrapping it in the conditional too. Correct me if I'm wrong 🫡